useCase/useCase.go

```diff
@@ -0,0 +1,40 @@
+package useCase
+
+import (
+	"strconv"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/utils"
+)
+
+func GetPokemonFromCSV(wantedIndex int) utils.PokemonData {
```

Comment:
Since there are multiple ways this function could fail, it should also return an error